### ISSUE TITLE
bug报告wechatpy.client.api.qrcode的def show(self, ticket)始终返回404

### ISSUE THREAD

**Issue by openpython** [wechatpy/wechatpy#113]
测了下，应该是ticket被requests二次编码

```
requests.get(
            url='https://mp.weixin.qq.com/cgi-bin/showqrcode',
            params={
                'ticket': ticket
            }
        )
```

**Comment by messense** [comment]
Fixed in commit @d30dd4db7f407e8f819516ab83384f0e24ac6d8b.

Thanks.

**Comment by messense** [comment]
v1.2.1 released with fixes.
